src/components/ActiveTeamsComponent.jsx

```diff
@@ -11,84 +9,69 @@ const ActiveTeamsComponent = ({
   handleDragStart,
   handleDragOver,
   handleDragDrop,
-}) => {
-  const myListRef = useRef(null);
-  const [isEmpty, setIsEmpty] = useState(true);
-
-  useEffect(() => {
-    if (myListRef.current === null) {
-      setIsEmpty(true);
-    } else if (myListRef.current.getElementsByTagName('li').length < 1) {
-      setIsEmpty(true);
-    } else {
-      setIsEmpty(false);
-    }
-  }, [teams]);
+}) => (
+  <div style={{ padding: '10px', border: '1px solid black' }}>
+    <h2>Active teams</h2>
+    <div
+      onDragOver={handleDragOver}
+      onDrop={handleDragDrop}
+      id="active"
+    >
+      {teams.length < 1 && (
+        <div
+          style={{ border: '2px solid green', width: '30vw' }}
+          id="active-1"
+        >
+          Drop here
+        </div>
+      )}
 
-  return (
-    <div style={{ padding: '10px', border: '1px solid black' }}>
-      <h2>Active teams</h2>
-      <div
-        onDragOver={handleDragOver}
-        onDrop={(e) => handleDragDrop(e)}
-        id='active'
-      >
-        {isEmpty && (
-          <div
-            style={{ border: '2px solid green', width: '30vw' }}
-            id='active-1'
-          >
-            Drop here
-          </div>
-        )}
-        {teams.length > 0 && (
-          <ul ref={myListRef}>
-            {teams.map(
-              (team, index) =>
-                team.active && (
-                  <li key={index} draggable>
-                    <button
-                      type='button'
-                      onDragStart={handleDragStart(index)}
-                      onDragOver={handleDragOver}
-                    >
-                      Drag
-                    </button>
-                    <input
-                      type='text'
-                      value={team.name}
-                      onChange={(e) => handleNameInput(e, index)}
```

Comment:
Two things here:
1. Don't shorten variable names. It makes it harder to read the code. And the build already takes care of minimizing the code size.
2. Try to avoid anonymous function calls in handlers. This functions are newly created on all renders. See Homepage.jsx to get the full picture.

Reply:
Noted.